spec/api/comment_thread_spec.rb

```diff
@@ -29,6 +29,16 @@ def thread_result(params)
             res["course_id"].should == "abc"
           }
         end
+        it "does not return standalone threads" do
+          [@threads["t1"], @threads["t2"], @threads["t3"]].each do |t|
+            t.course_id = "abc"
+            t.save!
+          end
+          @threads["t3"].context = :standalone
+          @threads["t3"].save!
+          rs = thread_result course_id: "abc", sort_order: "asc"
+          rs.length.should == 2
```

Comment:
I'd like this test to verify that the correct 2 threads are returned (instead of just 2 out of 3).